server/lib/util.py

```diff
@@ -593,15 +598,21 @@ def _get_highest_coverage_date(observation_entity_counts_by_date,
     max_years_to_check: Only consider entity counts going back this number of
       years
   """
-  # Get observation dates in descending order, and filter out dates in the
-  # future (to exclude erroneous data)
+  # Get observation dates in descending order
   todays_date = str(date.today())
```

Comment:
move todays_date to under the if condition?

Reply:
done